sympy/core/expr.py

```diff
@@ -3068,7 +3068,8 @@ def series(self, x=None, x0=0, n=6, dir="+", logx=None, cdir=0):
 
             try:
                 from sympy.simplify.radsimp import collect
-                return collect(s1, x) + o
+                from sympy.simplify.ratsimp import ratsimp
+                return collect(ratsimp(s1), x) + o
```

Comment:
I'm not sure about calling `ratsimp` here. If we want to canonicalise the form of the output then it should be done systematically and that really means using ringseries I think.